libs/velo-external-db-core/test/drivers/schema_matchers.ts

```diff
@@ -27,59 +36,34 @@ export const schemaHeadersListFor = (collections: any) => toHaveSchemas(collecti
 
 export const schemasWithReadOnlyCapabilitiesFor = (collections: any) => toHaveSchemas(collections, collectionToHaveReadOnlyCapability)
 
-
-const toHaveCollection = ( collections: any[], functionOnEachCollection: any, ...args: any ) =>  expect.objectContaining({
-    collection: collections.map((c: any) => functionOnEachCollection(c, args))
-})
-
-export const queryOperatorsFor = (fieldType: string): string[] => {
-    switch (fieldType) {
-        case 'text':
-        case 'url':
-            return [eq, ne, string_contains, string_begins, string_ends, include, gt, gte, lt, lte]
-        case 'number':
-            return [eq, ne, gt, gte, lt, lte, include]
-        case 'boolean':
-            return [eq]
-        case 'image':
-            return []
-        case 'object':
-            return [eq, ne]
-        case 'datetime':
-            return [eq, ne, gt, gte, lt, lte]
-    
-        default:
-            throw new Error(`${fieldType} - Unsupported field type`)
-    }
-
-}
-
-export const fieldInNewWixFormat = (field: any) => expect.objectContaining({
+export const fieldInNewWixFormat = (field: ResponseField) => expect.objectContaining({
```

Comment:
maybe rename it to matcher name (like the other comments), and in general `newWixFormat` doesn't sound to me like a good naming.

Reply:
removed new from the naming

Reply:
I removed the word `new` from the naming